dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -256,6 +256,7 @@
 			interrupts = <83 0>, <84 0>;
 			#pwm-cells = <2>;
 			clocks = <&syscon MCUX_BUS_CLK 0 0>;
+			nxp,prescaler = <7>;
```

Comment:
why is it defined at soc dts files? its value is really application specific (e.g. the pwm frequency you need to synthesize in your app), so it should be set in board files.

If register always has the same reset default, you can add such default to the property.

Reply:
I've followed how this is done for the rest vendors that have that property in the soc dts file.